packages/shared/src/write-archive/archive-file.ts

```diff
@@ -100,7 +100,7 @@ export class ArchiveFile {
 
   private removeSpecialChars(pathname: string) {
     // The storybook server seems to have a problem with percents in file names
-    return pathname.replace(/[%]/g, '');
+    return pathname.replace(/[%]|[:]/g, '');
```

Comment:
Minor but this is equivalent to:
```suggestion
    return pathname.replace(/[%:]/g, '');
```